src/main/kotlin/com/exactpro/th2/kafka/client/KafkaConnection.kt

```diff
@@ -59,11 +59,12 @@ class KafkaConnection(
     fun publish(message: RawMessage) {
         val alias = message.sessionAlias ?: error("Message '${message.id.logId}' does not contain session alias.")
         val value = message.body.toByteArray()
-        val bookConfig = config.booksConfigs[message.bookName] ?: config.defaultBookConfig
+
```

Comment:
Maybe extract this logic to a separate strategy object related to session alias. We can map strategy by SessionKey(book, alias, topic). Strategy can hold required values from config, alos, have logic to enrich RawMessage.Builder and build Kafka record